docs/redux.md

```diff
@@ -0,0 +1,118 @@
+# Redux configuration
+
+## What is Redux
+
+Flex Template for Web (FTW) is JavaScript single-page application (SPA). This means that the app
+needs to be able to render several different layouts (pages) depending on user interaction. State
+management is essential for this process. FTW needs to know if a user has been authenticated, if it
+has received relevant data for the current page, and so on.
+
+We use [Redux](https://redux.js.org/introduction) for state management on the application level. You
+should read more about Redux before you start modifying queries to Flex API or creating new Page
+level elements (unless you are modifying [a static page](static-pages.md)).
+
+In the following subtopics, we assume that you know the
+[basics of Redux](https://redux.js.org/basics) already.
+
+* [Containers: Pages + TopbarContainer](#containers-pages--topbarcontainer)
+* [Duck files](#duck-files)
+* [Setting up Redux](#setting-up-redux)
+* [Advanced Redux concepts: thunks](#advanced-redux-concepts-thunks)
+
+## Containers: Pages + TopbarContainer
+
+We have set up FTW so that pages are aware of Redux, but other components and forms are purely
+presentational components. This makes it easier to customize UI components - you don't need to be
```

Comment:
The "container" and "presentational" terms are not defined anywhere here and might be confusing. I don't think even the Redux basics docs mention them. Maybe link to Abramov's Medium post about them or add a simple explanation here. Those are quite crucial concepts to grok.